cli/endpoints/online/managed/binary-payloads/code/multi-file-to-json-score.py

```diff
@@ -0,0 +1,16 @@
+from azureml.contrib.services.aml_request import AMLRequest, rawhttp
```

Comment:
Is this package from SDK v1?
Do we have any ways to do this with SDK v2?

Reply:
Last thing here, theoretically an easy change but I had some problems with this recently.